extensions/arc/package.json

```diff
@@ -1047,6 +875,220 @@
             "when": "true"
           }
         ],
+        "agreement": [
+          {
+            "template": "%arc.agreement%",
+            "links": [
+              {
+                "text": "%microsoft.agreement.privacy.statement%",
+                "url": "https://go.microsoft.com/fwlink/?LinkId=853010"
+              },
+              {
+                "text": "%arc.agreement.postgres.terms.conditions%",
+                "url": "https://go.microsoft.com/fwlink/?linkid=2045708"
+              }
+            ],
+            "when": "true"
+          }
+        ]
+      }
+    ],
+    "resourceDeploymentSubTypes": [
+      {
+        "name": "azure-sql-mi",
+        "icon": "./images/miaa.svg",
+        "options": [
+          {
+            "name": "mi-type",
+            "displayName": "Resource Type",
```

Comment:
Localize

Reply:
Actually, this is not needed here. Removing it.